rero_ils/modules/documents/tasks.py

```
@@ -23,10 +23,39 @@
+    :returns: tuple (url, provider) or (None, None) if no cover was found.
+    """
+
+    isbns = [
+        identified_by.get("value")
+        for identified_by in document.get("identifiedBy", [])
+        if identified_by.get("type") == "bf:Isbn" and identified_by.get("value")
+    ]
+    for isbn in sorted(isbns):
+        try:
+            url, provider = get_thumbnail_url(isbn, cached=cached)
+        except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)